.github/workflows/branch_sync.yml

```diff
@@ -0,0 +1,103 @@
+name: Sync PR
+
+on:
+  schedule:
+    - cron: '33 15 * * 1-5'  # 15:33 UTC Mon-Fri
```

Comment:
If running on push removing cron would avoid the need to hardcode the 8.*.x branch name in the workflow and update it with each minor release.

```suggestion
```